src/midi/midi.ts

```diff
@@ -22,34 +22,32 @@ export default class MonologueController {
     if (this.demoMode) {
       return;
     }
+
     Object.keys(this.registeredParameters).map((id) => {
       const { parameter, callback } = this.registeredParameters[id];
+      const [message, messageParameter, value] = e.data; // eslint-disable-line @typescript-eslint/no-unused-vars
 
-      const [_, messageParameter, value] = e.data; // eslint-disable-line @typescript-eslint/no-unused-vars
+      // We only want to listen to param change messages
+      if(message !== PARAM_CHANGE_MESSAGE) return null;
 
       if (messageParameter !== parameter.ID) {
-        return;
+        return null;
       }
 
-      const knobValue = Math.round((value / 123) * 1027);
-      const threePoleValue = (value / 123) * 2;
-      const fourPoleValue = (value / 123) * 3;
+      return callback(parameter, value);
+    });
+  };
 
-      let finalValue;
-      if (parameter.type === ParameterType.LINEAR) {
-        finalValue = knobValue;
```

Comment:
Below looks really confusing, but I've actually just swapped a few functions around and it's got confused.